### PR TITLE
WIP: ENH: clean up strain_classification.py into classification.py

### PR DESCRIPTION
**Pull request by hduongtrong**
1. Create a function load_data to load the data and return X, Y for
        classification
2. Create a report_model_performance that print out CV accuracy of different
models when applying to our dataset

### CODE REVIEW

**Comment by hduongtrong** [comment]
It seems scikit-learn (sklearn) is not in our requirement.txt list, which causes Travis CI to fail. Can we put that to the list?

**Comment by dylandaniels** [comment]
@hduongtrong please add it to the file and commit!

**Comment by hduongtrong** [comment]
@dylandaniels thanks I just did that. I also add a few new features. I squashed both commits together and present it as one commit.

**Comment by changsiyao** [comment]
@hduongtrong 
Looks like you are having file not found issues. It might have something to do with how you set your path. 
Try: 
`import os as _os`
`from mousestyles import data_dir`
`path = _os.path.join(data_dir, 'all_features_mousedays_11bins.npy')`
`np.load(path)`
something like this. See if this helps with passing the checks.

**Comment by hduongtrong** [comment]
@changsiyao thanks a lot for the tip. I just did that and now it passed the travis test. Just curious any particular reason for doing "import os as _os", instead of just "import os"?

**Comment by changsiyao** [comment]
@hduongtrong Not sure why os was imported as _os. I just followed Dylan's script when I was adding to it earlier. @dylandaniels Any particular reason why we `import os as _os`?

File: mousestyles/classification.py [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+        "solver": "lsqr",
+        "shrinkage": "auto",
+    },
+    "QuadraticDiscriminantAnalysis": {
+        "reg_param": .1,
+    },
+}
+
+
+def load_data(data_path):
```
Comment (peterasujan) [inline mousestyles/classification.py]: @hduongtrong this function should probably be moved to the data module (or, it may be a duplicate of something that's already there).

Comment (hduongtrong) [inline mousestyles/classification.py]: @peterasujan I don't think it's in other files yet (?), but let me move it to data_utils.

**Comment by dylandaniels** [comment]
@hduongtrong has anyone reviewed this? Why is this set to MRG?

**Comment by hduongtrong** [comment]
@dylandaniels I changed that back to WIP.
